src/layman/common/rest.py

```diff
@@ -105,16 +103,29 @@ def setup_post_access_rights(request_form, kwargs, actor_name):
 
 
 def get_publications(publication_type, user, request_args):
+    known_order_by_values = [consts.ORDER_BY_TITLE, consts.ORDER_BY_FULL_TEXT, ]
 
     full_text_filter = None
     if consts.FILTER_FULL_TEXT in request_args:
         full_text_filter = prime_db_schema_util.to_tsquery_string(request_args.get(consts.FILTER_FULL_TEXT)) or None
 
     order_by_list = []
+    if consts.ORDER_BY_PARAM in request_args:
+        order_by_value = request_args.get(consts.ORDER_BY_PARAM)
+        if order_by_value not in known_order_by_values:
+            raise LaymanError(48, f'Known values for "order_by" are: {known_order_by_values}')
+
+        if order_by_value == consts.ORDER_BY_FULL_TEXT and not full_text_filter:
+            raise LaymanError(49, f'order_by parameter "{consts.ORDER_BY_FULL_TEXT}" can be '
```

Comment:
Change 49 to 48

Rewording: `f'Value "{consts.ORDER_BY_FULL_TEXT}" of parameter "order_by" can be used only if "{consts.FILTER_FULL_TEXT}" parameter is set."`